steam_wrapper/steam_wrapper.py

```diff
@@ -96,7 +97,7 @@ def check_bad_filesystem_entries(entries):
                  "host",
                  os.path.expandvars("/var/home/$USER"),
                  os.path.expandvars("/home/$USER")]
-    bad_topdirs = ["xdg-config", "xdg-data", "xdg-cache"]
+    bad_topdirs = ["xdg-data", "xdg-cache", "home", "host"]
```

Comment:
AFAIK `home` and `host` are special values that can't be used in paths (e.g. `--filesystem=home/.config` won't work). Needs confirmation.

Reply:
They can, this was changed in a recent PR.

Reply:
Ah, okay, I thought it was both.